config/initializers/content_security_policy.rb

```diff
@@ -1 +1,12 @@
-GovukContentSecurityPolicy.configure
+GovukContentSecurityPolicy.configure do |policy|
+  # Ensures the ability to use inline JavaScript without protections. This is
+  # required for compatibility with govuk_admin_template which both uses script
+  # tags without nonces and uses jQuery 1.x which requires unsafe-inline in
+  # some browsers (Firefox is one)
+  script_policy_with_unsafe_inline = (policy.script_src + ["'unsafe-inline'"]).uniq
```

Comment:
I'm assuming the double quotes here is intended and not a typo, but just checking?

Reply:
Yup, thanks. They're intentional - we need unsafe-inline in quotes.